repository/n.json

```diff
@@ -111,6 +111,17 @@
 				}
 			]
 		},
+		{
+			"name": "NativeScript Core XML Syntax",
```

Comment:
Language support packages can just be named after the language, without suffixes like "syntax", "language", "highlighting" etc. In this case I would lean towards "NativeScript XML"?

Reply:
NativeScript is a mobile framework with multiple flavors (NativeScript Vue, Svelte, etc). NativeScript Core is a flavor itself, hence the `NativeScript Core` reference.

NativeScript Core consists of JS, CSS, and XML files. In XML, there is a custom expression syntax where attribute values are enclosed in double curly braces (e.g. `{{ myVariable }}`), that is why I focus on syntax.

Reply:
Alright, makes sense then 👍🏻